print/src/main.rs

```diff
@@ -94,20 +100,74 @@ const READ_BUFFER_SIZE: usize = 1024;
 
 #[tokio::main]
 async fn main() -> anyhow::Result<()> {
+    let opts = Opts::parse();
+
+    if opts.version {
+        return print_version();
+    }
+
+    // We create the source outside of the run command since recreating the stdin looses us some frames
+    let mut source = match opts.command.clone() {
+        None | Some(Command::Stdin) => Source::stdin(),
+        Some(Command::Tcp { host, port }) => Source::tcp(host, port).await?,
+    };
+
+    if opts.watch_elf {
+        let (tx, mut rx) = tokio::sync::mpsc::channel(1);
+
+        let mut watcher = RecommendedWatcher::new(
+            move |res| {
+                futures::executor::block_on(async {
+                    tx.send(res).await.unwrap();
+                })
```

Comment:
Would [tokio::runtime::Handle.current()](https://docs.rs/tokio/latest/tokio/runtime/struct.Handle.html#method.current) work here? Then we could get rid of the `futures` dependency.
```suggestion
                tokio::runtime::Handle::current().block_on(async {
                    tx.send(res).await.unwrap();
                })
```

Reply:
It's probably even easier to use `tx.blocking_send` instead, that way there is no need for any async context. I'll try that later today